services/QuillLMS/app/workers/admin_dashboard/made_school_admin_change_school_email_worker.rb

```diff
@@ -11,7 +11,9 @@ def perform(user_id, admin_user_id, new_school_id, existing_school_id)
 
     return unless user && admin_name && new_school && existing_school
 
-    user.mailer_user.send_admin_dashboard_made_school_admin_change_school_email(admin_name, new_school, existing_school)
+    if SchoolsAdmins.where(user_id: user_id).count == 1
```

Comment:
This conditional clause violates the law of demeter.  We can fix that by adding a boolean method to user.rb

```
def is_admin_for_one_school?
  schools_admins.count == 1
end
```
then the conditional becomes:

`if user.is_admin_for_one_school?`

Reply:
Cool, I'll make that change!